### PR TITLE
update fishing pets

### PR DESCRIPTION
**Pull request by catgirlseraid**
ammonite scc im unsure about, as it's based off hotm level.
assuming hotm 0 it's 0.05-5 so I've changed it to this for now as it seems to be standard for other pets such as golden dragon and gold collection but im happy to revert if wanted <3

ammonite
yeti
blue whale
dolphin
flying fish
megalodon
reindeer
squid

### CODE REVIEW

**Review by jani270 — CHANGES_REQUESTED** [review]
grafik
lmao (account doesnt have hotm unlocked)*

**Comment by catgirlseraid** [comment]
changed ammonite from +0.05% -> +0.06% and +5% -> +6%

File: constants/petnums.json [hunk 1]
```diff
@@ -4793,7 +4793,7 @@
     },
     "RARE": {
       "1": {
-        "otherNums": [
+        "otherNums":  [
```
Comment (jani270) [inline constants/petnums.json:4796]: Whats up with this?

**Review by jani270 — APPROVED** [review]
surely this is all fine

**Pull request merged**